ee/clickhouse/models/cohort.py

```diff
@@ -47,7 +47,7 @@ def format_person_query(cohort: Cohort, index: int, **kwargs) -> Tuple[str, Dict
 
     if not groups:
         # No person can match a cohort that has no match groups
-        return "0 = 1", {}
+        return "8 = 9", {}
```

Comment:
?

Reply:
I know this looks slightly weird. 😅 It's because we have these guaranteed false inequalities to filter things out in various places. Most commonly `0 = 1` – not just `0` since that doesn't work. But at least using inequalities, we can uniquely identify which function they were added in by just using different numbers, which doesn't cost us anything while making debugging _a tiny bit_ easier.

Reply:
The thing this makes worse is grepping for these inequalities.

How about `0 = 1X` where X is increasing instead?

Reply:
Sure, that's even better!